docs/src/build-from-source.md

```diff
@@ -30,7 +30,7 @@ git submodule update --init
 ./scripts/setup/ubuntu/install_deps.sh
 ./scripts/setup/ubuntu/install_cbmc.sh
 ./scripts/setup/ubuntu/install_viewer.sh
-# If you haven't already:
+# If you haven't already (or from [rustup](https://rustup.rs/)):
```

Comment:
Note that this is inside a code block, so the link won't be rendered. I.e.: The text will show up as is:

https://github.com/model-checking/kani/blob/7a2dab3c9eec8ea772c5f3e3262d6cf130f1dd0f/docs/src/build-from-source.md#install-dependencies-on-ubuntu